client/components/plans/plan-price/index.jsx

```diff
@@ -23,8 +23,9 @@ const PlanPrice = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
-			const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
-			formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
+			const currencySymbol = formattedPrice.slice( 0, 1 );
```

Comment:
<img width="765" alt="screen shot 2016-06-13 at 12 35 48 pm" src="https://cloud.githubusercontent.com/assets/1270189/16020536/6f686afa-3163-11e6-9381-865ca5ea3c82.png">
It looks like we localize incorrectly for euros, but let me find a more robust check.